src/components/macroview/rightPanel/EditArea.tsx

```diff
@@ -12,6 +12,7 @@ import SystemEventActionForm from './editForms/SystemEventActionForm'
 export default function EditArea() {
   const selectedElement = useSelectedElement()
   const { selectedElementId } = useMacroContext()
+  useColorModeValue('primary-light.500', 'primary-dark.500')
```

Comment:
🤔 This isn't being used, it's not even being put into a variable

Reply:
Oops, fixed: [9cac5c8](https://github.com/WootingKb/wooting-macros/pull/194/commits/9cac5c86f53aacbdb43852fd5febf478f3cde0eb)